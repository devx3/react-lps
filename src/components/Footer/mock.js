export const mock = {
    html: 'Thanks little bastard. <a href="http://localhost">DevX3</a>.',
};
