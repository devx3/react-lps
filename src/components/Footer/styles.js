import styled, { css } from 'styled-components';
import { Container as TextComponent } from '../TextComponent/styles';
import { Container as SectionContainer } from '../SectionContainer/styles';

export const Container = styled.div`
    text-align: center;
    ${({ theme }) => css`
        border-top: 1px solid ${theme.colors.mediumGrey};
        a {
            text-decoration: none;
        }

        & ${TextComponent} p {
            font-size: ${theme.fonts.sizes.xsmall};
        }

        & ${SectionContainer} {
            padding-top: 0;
            padding-bottom: 0;
        }
    `};
`;
