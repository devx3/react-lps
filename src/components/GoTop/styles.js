import styled, { css } from 'styled-components';

export const Container = styled.a`
    ${({ theme }) => css`
        background: ${theme.colors.secondaryColor};
        color: ${theme.colors.primaryColor};
        display: flex;
        align-items: center;
        justify-content: center;
        position: fixed;
        width: 4rem;
        height: 4rem;
        bottom: 2rem;
        right: 2rem;
        z-index: 6;
        cursor: pointer;
    `}
`;
