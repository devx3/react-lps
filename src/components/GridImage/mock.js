export const mock = {
    title: 'Hello GridImage',
    description: `Veniam velit magna nulla ex veniam sunt est.
    Consectetur ea aliquip do in. Qui amet non mollit elit proident
    aute adipisicing labore cillum. Non labore commodo minim labore
    ad sint eiusmod mollit anim cillum culpa officia cupidatat
    Lorem. Duis excepteur non ut consectetur adipisicing veniam sint.
    Aliquip ex excepteur non deserunt Lorem. Eu excepteur irure
    labore aliquip non aute sint sunt elit laboris occaecat ad veniam.`,
    grid: [
        {
            altText: 'Image 1',
            srcImage: 'https://source.unsplash.com/random/800x800?r=1',
        },
        {
            altText: 'Image 2',
            srcImage: 'https://source.unsplash.com/random/800x800?r=2',
        },
        {
            altText: 'Image 3',
            srcImage: 'https://source.unsplash.com/random/800x800?r=3',
        },
        {
            altText: 'Image 4',
            srcImage: 'https://source.unsplash.com/random/800x800?r=4',
        },
        {
            altText: 'Image 5',
            srcImage: 'https://source.unsplash.com/random/800x800?r=5',
        },
        {
            altText: 'Image 6',
            srcImage: 'https://source.unsplash.com/random/800x800?r=6',
        },
    ],
};
