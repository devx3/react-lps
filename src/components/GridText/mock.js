export const mock = {
    background: false,
    title: 'Um title qualquer',
    description: `Quis voluptate culpa dolor mollit minim non et.
    Veniam dolore amet exercitation non labore esse laborum. Aute
    nulla voluptate exercitation consectetur. Excepteur non et
    amet commodo irure. Eiusmod veniam laborum deserunt enim quis
    laborum cupidatat deserunt adipisicing id deserunt aute mollit
    culpa.`,
    grid: [
        {
            title: 'Title 1',
            description: `Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.`,
        },
        {
            title: 'Title 2',
            description: `Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.`,
        },
        {
            title: 'Title 3',
            description: `Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.`,
        },
    ],
};
