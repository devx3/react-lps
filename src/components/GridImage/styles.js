import styled, { css } from 'styled-components';
import { Container as TextContainer } from '../TextComponent/styles';

export const Container = styled.div`
    ${({ theme }) => css`
        ${TextContainer} {
            margin-bottom: ${theme.spacings.xhuge};
        }
    `}
`;

export const Grid = styled.div`
    ${({ theme }) => css`
        counter-reset: grid-counter;
        display: grid;
        grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
        gap: ${theme.spacings.large};
    `}
`;

export const GridItem = styled.div`
    ${() => css`
        overflow: hidden;
    `}
`;

export const Image = styled.img`
    ${() => css`
        width: 100%;
        transition: all 300ms ease-in-out;

        &:hover {
            transform: scale(1.2) rotate(5deg);
        }
    `}
`;
