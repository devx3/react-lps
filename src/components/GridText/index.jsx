import * as Styled from './styles.js';
import P from 'prop-types';
import { SectionBackground } from '../SectionBackground';
import { Heading } from '../Heading';
import { TextComponent } from '../TextComponent';

const GridText = ({
    title,
    description,
    grid = [],
    background = false,
    sectionId = '',
}) => {
    return (
        <SectionBackground background={background} sectionId={sectionId}>
            <Styled.Container>
                <Heading as="h2" size="huge" uppercase darkMode={background}>
                    {title}
                </Heading>
                <TextComponent>{description}</TextComponent>
                <Styled.Grid>
                    {grid.map((item) => (
                        <Styled.GridItem key={item.title}>
                            <Heading
                                as="h3"
                                darkMode={background}
                                size="medium"
                            >
                                {item.title}
                            </Heading>
                            <TextComponent>{item.description}</TextComponent>
                        </Styled.GridItem>
                    ))}
                </Styled.Grid>
            </Styled.Container>
        </SectionBackground>
    );
};

GridText.propTypes = {
    background: P.bool,
    title: P.string.isRequired,
    description: P.string.isRequired,
    grid: P.arrayOf(
        P.shape({
            title: P.string.isRequired,
            description: P.string.isRequired,
        }),
    ),
    sectionId: P.string,
};

export default GridText;
