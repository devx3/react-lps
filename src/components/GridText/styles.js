import styled, { css } from 'styled-components';
import { Title as HeadingContainer } from '../Heading/styles';
import { Container as TextContainer } from '../TextComponent/styles';

export const Container = styled.div`
    ${({ theme }) => css`
        ${TextContainer} {
            margin-bottom: ${theme.spacings.xhuge};
        }
    `}
`;

export const Grid = styled.div`
    ${({ theme }) => css`
        counter-reset: grid-counter;
        display: grid;
        grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
        gap: ${theme.spacings.large};

        @media ${theme.media.ltMedia} {
            grid-template-columns: 1fr;
        }
    `}
`;

export const GridItem = styled.div`
    ${({ theme }) => css`
        ${HeadingContainer} {
            position: relative;
            left: 5rem;
            margin-bottom: 15px;
        }

        ${HeadingContainer}::before {
            counter-increment: grid-counter;
            content: counter(grid-counter);
            position: absolute;
            top: -2rem;
            font-size: ${theme.fonts.sizes.huge};
            left: -5rem;
            transform: rotate(-10deg);
        }
    `}
`;
