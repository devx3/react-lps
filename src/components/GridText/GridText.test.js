import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import GridText from '.';
import { mock } from './mock';

describe('<GridText />', () => {
    it('Should render with default values', () => {
        const { container } = renderTheme(<GridText {...mock} />);
        expect(container).toMatchSnapshot();
    });

    it('Should render with grid', () => {
        const { container } = renderTheme(
            <GridText {...mock} grid={mock.grid} />,
        );
        expect(container).toMatchSnapshot();
    });

    it('Should render with background', () => {
        const { container } = renderTheme(
            <GridText {...mock} background={true} />,
        );
        expect(container).toMatchSnapshot();
    });
    it('Should render with no background', () => {
        const { container } = renderTheme(
            <GridText {...mock} background={undefined} />,
        );
        expect(container).toMatchSnapshot();
    });

    it('Should render with no grid', () => {
        const { container } = renderTheme(
            <GridText {...mock} grid={undefined} />,
        );
        expect(container).toMatchSnapshot();
    });
});
