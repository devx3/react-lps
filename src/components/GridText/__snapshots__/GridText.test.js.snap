// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridText /> Should render with background 1`] = `
.c4 {
  color: #FFFFFF;
  font-size: 5.6rem;
  text-transform: uppercase;
}

.c9 {
  color: #FFFFFF;
  font-size: 2.4rem;
  text-transform: none;
}

.c6 {
  font-size: 1.6rem;
}

.c2 .c5 {
  margin-bottom: 6.4rem;
}

.c7 {
  counter-reset: grid-counter;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
}

.c8 .c3 {
  position: relative;
  left: 5rem;
  margin-bottom: 15px;
}

.c8 .c3::before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  top: -2rem;
  font-size: 5.6rem;
  left: -5rem;
  -webkit-transform: rotate(-10deg);
  -ms-transform: rotate(-10deg);
  transform: rotate(-10deg);
}

.c0 {
  background: #FFFFFF;
  color: #0A1128;
  background: #0A1128;
  color: #FFFFFF;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem !important;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

@media (max-width:768px) {
  .c4 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c9 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c7 {
    grid-template-columns: 1fr;
  }
}

<div>
  <div
    class="c0"
    id="id-6267-506403265062"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <h2
          class="c3 c4"
        >
          Um title qualquer
        </h2>
        <div
          class="c5 c6"
        >
          Quis voluptate culpa dolor mollit minim non et.
    Veniam dolore amet exercitation non labore esse laborum. Aute
    nulla voluptate exercitation consectetur. Excepteur non et
    amet commodo irure. Eiusmod veniam laborum deserunt enim quis
    laborum cupidatat deserunt adipisicing id deserunt aute mollit
    culpa.
        </div>
        <div
          class="c7"
        >
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 1
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 2
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 3
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<GridText /> Should render with default values 1`] = `
.c4 {
  color: #0A1128;
  font-size: 5.6rem;
  text-transform: uppercase;
}

.c9 {
  color: #0A1128;
  font-size: 2.4rem;
  text-transform: none;
}

.c6 {
  font-size: 1.6rem;
}

.c2 .c5 {
  margin-bottom: 6.4rem;
}

.c7 {
  counter-reset: grid-counter;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
}

.c8 .c3 {
  position: relative;
  left: 5rem;
  margin-bottom: 15px;
}

.c8 .c3::before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  top: -2rem;
  font-size: 5.6rem;
  left: -5rem;
  -webkit-transform: rotate(-10deg);
  -ms-transform: rotate(-10deg);
  transform: rotate(-10deg);
}

.c0 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem !important;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

@media (max-width:768px) {
  .c4 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c9 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c7 {
    grid-template-columns: 1fr;
  }
}

<div>
  <div
    class="c0"
    id="id-6478-419044838526"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <h2
          class="c3 c4"
        >
          Um title qualquer
        </h2>
        <div
          class="c5 c6"
        >
          Quis voluptate culpa dolor mollit minim non et.
    Veniam dolore amet exercitation non labore esse laborum. Aute
    nulla voluptate exercitation consectetur. Excepteur non et
    amet commodo irure. Eiusmod veniam laborum deserunt enim quis
    laborum cupidatat deserunt adipisicing id deserunt aute mollit
    culpa.
        </div>
        <div
          class="c7"
        >
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 1
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 2
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 3
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<GridText /> Should render with grid 1`] = `
.c4 {
  color: #0A1128;
  font-size: 5.6rem;
  text-transform: uppercase;
}

.c9 {
  color: #0A1128;
  font-size: 2.4rem;
  text-transform: none;
}

.c6 {
  font-size: 1.6rem;
}

.c2 .c5 {
  margin-bottom: 6.4rem;
}

.c7 {
  counter-reset: grid-counter;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
}

.c8 .c3 {
  position: relative;
  left: 5rem;
  margin-bottom: 15px;
}

.c8 .c3::before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  top: -2rem;
  font-size: 5.6rem;
  left: -5rem;
  -webkit-transform: rotate(-10deg);
  -ms-transform: rotate(-10deg);
  transform: rotate(-10deg);
}

.c0 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem !important;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

@media (max-width:768px) {
  .c4 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c9 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c7 {
    grid-template-columns: 1fr;
  }
}

<div>
  <div
    class="c0"
    id="id-4172-924080321721"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <h2
          class="c3 c4"
        >
          Um title qualquer
        </h2>
        <div
          class="c5 c6"
        >
          Quis voluptate culpa dolor mollit minim non et.
    Veniam dolore amet exercitation non labore esse laborum. Aute
    nulla voluptate exercitation consectetur. Excepteur non et
    amet commodo irure. Eiusmod veniam laborum deserunt enim quis
    laborum cupidatat deserunt adipisicing id deserunt aute mollit
    culpa.
        </div>
        <div
          class="c7"
        >
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 1
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 2
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 3
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<GridText /> Should render with no background 1`] = `
.c4 {
  color: #0A1128;
  font-size: 5.6rem;
  text-transform: uppercase;
}

.c9 {
  color: #0A1128;
  font-size: 2.4rem;
  text-transform: none;
}

.c6 {
  font-size: 1.6rem;
}

.c2 .c5 {
  margin-bottom: 6.4rem;
}

.c7 {
  counter-reset: grid-counter;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
}

.c8 .c3 {
  position: relative;
  left: 5rem;
  margin-bottom: 15px;
}

.c8 .c3::before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  top: -2rem;
  font-size: 5.6rem;
  left: -5rem;
  -webkit-transform: rotate(-10deg);
  -ms-transform: rotate(-10deg);
  transform: rotate(-10deg);
}

.c0 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem !important;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

@media (max-width:768px) {
  .c4 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c9 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c7 {
    grid-template-columns: 1fr;
  }
}

<div>
  <div
    class="c0"
    id="id-9477-080368625155"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <h2
          class="c3 c4"
        >
          Um title qualquer
        </h2>
        <div
          class="c5 c6"
        >
          Quis voluptate culpa dolor mollit minim non et.
    Veniam dolore amet exercitation non labore esse laborum. Aute
    nulla voluptate exercitation consectetur. Excepteur non et
    amet commodo irure. Eiusmod veniam laborum deserunt enim quis
    laborum cupidatat deserunt adipisicing id deserunt aute mollit
    culpa.
        </div>
        <div
          class="c7"
        >
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 1
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 2
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Title 3
            </h3>
            <div
              class="c5 c6"
            >
              Quis voluptate culpa dolor mollit minim non et.
            Veniam dolore amet exercitation non labore esse laborum. Aute
            nulla voluptate exercitation consectetur. Excepteur non et
            amet commodo irure. Eiusmod veniam laborum deserunt enim quis
            laborum cupidatat deserunt adipisicing id deserunt aute mollit
            culpa.
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<GridText /> Should render with no grid 1`] = `
.c3 {
  color: #0A1128;
  font-size: 5.6rem;
  text-transform: uppercase;
}

.c5 {
  font-size: 1.6rem;
}

.c2 .c4 {
  margin-bottom: 6.4rem;
}

.c6 {
  counter-reset: grid-counter;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
}

.c0 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  max-width: 120rem !important;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

@media (max-width:768px) {
  .c3 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c6 {
    grid-template-columns: 1fr;
  }
}

<div>
  <div
    class="c0"
    id="id-4061-6228667964215"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <h2
          class="c3"
        >
          Um title qualquer
        </h2>
        <div
          class="c4 c5"
        >
          Quis voluptate culpa dolor mollit minim non et.
    Veniam dolore amet exercitation non labore esse laborum. Aute
    nulla voluptate exercitation consectetur. Excepteur non et
    amet commodo irure. Eiusmod veniam laborum deserunt enim quis
    laborum cupidatat deserunt adipisicing id deserunt aute mollit
    culpa.
        </div>
        <div
          class="c6"
        />
      </div>
    </div>
  </div>
</div>
`;
